src/compiler/commandLineParser.ts

```diff
@@ -1331,7 +1331,14 @@ namespace ts {
             description: Diagnostics.Delete_the_outputs_of_all_projects,
             type: "boolean",
             defaultValueDescription: false,
-        }
+        },
+        {
+            name: "solutionDiagnostics",

```

Comment:
What does "solution" mean in this context?

Reply:
its like "client-web/tsconfig.json".. Its like instead of or in addition to per project perf statistics that get reported, add them together and report them and also add bunch of tsc -build related perf numbers...